tests/test_parse_digest_host_url.py

```diff
@@ -0,0 +1,39 @@
+from flask_static_digest import parse_digest_host_url
```

Comment:
Thoughts on replicating all of these tests without using pytest, following the same patterns used in the existing test suite?